.github/workflows/countline.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python
+
+import os
+import glob
+
+#get current working dir, set count, and select file delimiter
+path = os.getcwd()
+#os.chdir(path)
+
+
+#parses through files and saves to a dict
+names={}
+pathName = path + '/**/*.dart'
```

Comment:
Use [argparse](https://docs.python.org/3/library/argparse.html) for specifying the directory with an optional default.

Use [argparse](https://docs.python.org/3/library/argparse.html) for specifying the line count with a default value

Reply:
os.getcwd() used